tests/conditional_compilation/tools/infer_tool.py

```diff
@@ -11,27 +11,37 @@
 import sys
 import os
 import numpy as np
-from openvino.inference_engine import IENetwork, IECore
+from openvino.inference_engine import IECore
 
 log.basicConfig(format="[ %(levelname)s ] %(message)s", level=log.INFO, stream=sys.stdout)
 
 
+def input_preparation(net):
+    """
+    Function to prepare reproducible from run to run input data
+    :param net: IENetwork read object
+    :return: Dict containing ndarray of ones with the dimension of net input items
+    """
+
+    feed_dict = {}
+    for layer, name in net.inputs.items():
```

Comment:
please, update `layer, name` to `layer_name, layer_data` (or something like that). Second value contains layer info, not a name